blog/_posts/2018-11-09-dvteam-farewell

```diff
@@ -0,0 +1,134 @@
+---
+title: DV Team Farewell
```

Comment:
Hi @dianavile . Thanks for the PR. The file should be markdown file. You forgot the extension here.